docs/control-structures.rst

```diff
@@ -145,6 +145,39 @@ throws an exception or goes out of gas.
     external functions happen after any changes to state variables in your contract
     so your contract is not vulnerable to a reentrancy exploit.
 
+.. warning::
+    External function pointers only take memory arguments, even if the function signature uses calldata.
+    This is because for external function pointers, you are only interested in the calling side and not in the called side.
+    For the caller, it does not make sense to even distinguish `function (string calldata) external` and `function (string memory) external`.
```

Comment:
It's RST, not Markdown. You need double backticks:
```suggestion
    For the caller, it does not make sense to even distinguish ``function (string calldata) external`` and ``function (string memory) external``.
```

Reply:
Ok.